src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java

```diff
@@ -61,6 +68,17 @@
 @Restricted(NoExternalUse.class)
 public class ContentMappings extends ManagementLink implements Saveable, Iterable<ContentMapping> {
 
+    /**
+     * Name of the file containing additional user-provided stop words.
+     */
+    private static final String ADDITIONAL_STOP_WORDS_FILENAME = "additional-stop-words.txt";
+
+    /**
+     * Property to set to add <b>additional</b> stop words.
+     * The location should point to a line separated file containing words. Each line is treated as a word. 
+     */
+    static final String ADDITIONAL_STOP_WORDS_PROPERTY = ContentMappings.class.getName()+".additionalStopWordsFile";
```

Comment:
is it used externally or it can be private @Dohbedoh ?

```suggestion
    private static final String ADDITIONAL_STOP_WORDS_PROPERTY = ContentMappings.class.getName()+".additionalStopWordsFile";
```